src/library-authoring/LibraryLayout.tsx

```diff
@@ -44,7 +40,7 @@ const LibraryLayout = () => {
         </>
       </SidebarProvider>
     </LibraryProvider>
-  ), [location.pathname]);
+  ), []);
```

Comment:
This fixes the issue where the `LibraryProvider` was being remounted when the `location.pathname` changes.